### DISCUSSION TITLE
'Using Flux on Azure' documentation is out-of-date: Helm OCI registry compatibility?

### DISCUSSION THREAD

**Discussion by stevendick-work** [fluxcd/flux2#2841] [General] [answered]
The documentation states that Helm OCI registries are not supported, but isn't OCI support now available since Flux 0.31.0?

**Comment by hiddeco** [comment, marked as answer]
This is indeed available since Flux 0.31.0, and the note is outdated. You can find an example of an OCI HelmRepository object here: https://fluxcd.io/docs/components/source/helmrepositories/#helm-oci-repository

**Reply by souleb** [reply]
There is an ongoing PR to update the doc.
